### PR TITLE
option to add few products with diffrenet options

### PR DESCRIPTION
**Pull request by MoishyHinkis**

### CODE REVIEW

File: src/Http/Controllers/CartItemController.php [hunk 1]
```diff
@@ -16,11 +16,21 @@ public function store(Request $request)
...
 
         $product = $request->product_type::findOrFail($request->product_id);
-        Cart::add($product, $request->quantity ?? 1, $request->options);
+
+        if ($request->quantity > 1 &&  $request->multi) {
+            $multiOptions = $request->options;
+            foreach ($request->options[$request->multi] as  $value) {
+                $multiOptions[$request->multi] = $value;
+                Cart::add($product, 1, $multiOptions);
+            }
```
Comment (Yiddishe-Kop) [inline src/Http/Controllers/CartItemController.php:30]: I think it should be something like this:
```php
foreach ($request->options as $options) {
    Cart::add($product, 1, $options);
}
```

**Review by Yiddishe-Kop — CHANGES_REQUESTED** [review]

File: src/Http/Controllers/CartItemController.php [hunk 2]
```diff
@@ -16,11 +16,21 @@ public function store(Request $request)
             'product_type' => 'required|string',
             'product_id' => 'required',
             'quantity' => 'nullable|numeric',
-            'options' => 'nullable|array'
+            'options' => 'nullable|array',
+            'multi' => 'nullable|string'
```
Comment (Yiddishe-Kop) [inline src/Http/Controllers/CartItemController.php:20]: Shouldn't it be a `boolean`?
```php
            'multi' => 'nullable|boolean'
```

Comment (MoishyHinkis) [reply src/Http/Controllers/CartItemController.php:20]: basically itt shouldn't be at all, but i didn't know how you used this package in other projects, or how you will want to use it in future, so i did it like this

File: src/Http/Controllers/CartItemController.php (hunk 1 above)
Comment (MoishyHinkis) [reply src/Http/Controllers/CartItemController.php:30]: i don't think it's work.
did you tryed this and it worked?

**Pull request closed without merging**